### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by Vaibhavpratapsingh22**

### CODE REVIEW

**Review by udeysingh95 — CHANGES_REQUESTED** [review]
Make it crisp. This is too long.

**Review by SaarthakDeshmukh — APPROVED** [review]
I see the said changes are been made. Good to go from my side. :white_check_mark:

**Review by udeysingh95 — CHANGES_REQUESTED** [review]
There are too many topics in this file at the moment - Think of what we actually practise - and somethings maybe -which we want to have in coming days or weeks. 
And keep only those. I think 10-20 lines should be more than enough to cover all that. 
At the moment - it is very detailed but its not ours and has too much irrelevant info.

**Review by SaarthakDeshmukh — COMMENTED** [review]
It's crisp and clear. For now, this is okay, but later on, when there are more people we need to add a few more specific details, like what kind of behavior is not tolerated. For ex. Changing someone else's file or structure of the files. and things like that.

**Comment by Vaibhavpratapsingh22** [comment]
> It's crisp and clear. For now, this is okay, but later on, when there are more people we need to add a few more specific details, like what kind of behavior is not tolerated. For ex. Changing someone else's file or structure of the files. and things like that.  

True @SaarthakDeshmukh i will update it time to time.

**Comment by udeysingh95** [comment]
It could be also added - new contributors are only allowed to "watch" in the beginning and commit nearly nothing. Also, maybe something around our whatsapp group. what do you say?
The language of this document is very official - and that is not the style of our work, nor team.

**Comment by Vaibhavpratapsingh22** [comment]
Okay, @udeysingh95 I will work on it.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,15 +1,19 @@
-# Contributor Covenant Code of Conduct
+# Contributor Code of Conduct
```
Comment (SaarthakDeshmukh) [inline CODE_OF_CONDUCT.md:2]:
```suggestion
A code of conduct is a set of rules outlining the norms, rules, and responsibilities or proper practices of an individual party or an organization.
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+We work in a very simple manner so that it is easy to understand by contributors new and existing both. We all stay connected through social networks like whatsapp, zoom, google meets, skype. On these platforms we do open discussions and share our ideas freely and resolve the problems collectively with the help and support of each other.  
+
+## Our Responsibilities
+Existing contributors have the right and responsibility to  edit comments, commits, code, issues, and other contributions. 
+
+## New Contributor Rights
+New contributors can read, ask questions and suggest better ideas if any.
+
+## Enforcement
+For any kind of support, complaint and query mail us at vaibhavpratapsingh22@gmail.com.
```
Comment (udeysingh95) [inline CODE_OF_CONDUCT.md:19]:
```suggestion
```

**Review by udeysingh95 — CHANGES_REQUESTED** [review]

**Review by udeysingh95 — APPROVED** [review]

**Pull request merged**